app/components/chart/footer.js

```diff
@@ -74,28 +77,41 @@ var TidelineFooter = React.createClass({
             onChange={this.props.onClickLines} /> Lines
         </label>
       </div>
-      );
-    
+    );
 
-    
-    var rightSide = this.props.chartType === 'weekly' ? showValues :
-      this.props.chartType === 'modal' ? modalOpts : null;
-    
+    var rightSide = null;
+    var bgDataToggle = null;
+
+    if (this.props.chartType === 'weekly') {
+      rightSide = showValues;
+    } else if (this.props.chartType === 'modal') {
```

Comment:
Is this `else` necessary, or could it just be `<newline> if (this.props.chartType === 'modal') {`?